README.md

```diff
@@ -0,0 +1,438 @@
+
+Besides PDF parsing and writing PoDoFo includes also very simple classes to create your own PDF files. All classes are documented so it is easy to start writing your own application using PoDoFo.
+
+PoDoFo is primarily useful for applications that wish to do lower level manipulation of PDF, such as extracting content or merging files. It's also useful if your application has specific requirements for it's PDF output that more general output-oriented libraries like Cairo cannot satisfy. Canvas/drawing support is currently very limited in PoDoFo, so for pure output tasks a library like Cairo will be more suitable. PoDoFo cannot render PDF, so you should look at a library like Poppler for that.
+
+See:
+
+- [http://cairographics.org/](http://cairographics.org/)
+- [http://poppler.freedesktop.org/](http://poppler.freedesktop.org/)
+
+The PoDoFo tools are simple tools build around the PoDoFo library. These tools are first of all examples on how to use the PoDoFo library in your own projects. But secondly they offer also features for working with PDF files. More tools will come with future release and the existing tools will gain more features. The following tools are currently available:
+
+- **podofoencrypt** - Encrypts any PDF and allows to set PDF permissions..
+- **podofoimgextract** - Extracts all images from a given PDF file.
+- **podofoimpose** - A powerful PDF [imposition](http://en.wikipedia.org/wiki/Imposition) tool. It places pages from one or more source PDFs onto pages of a new PDF, applying scaling and positioning. If you have lua5.1 development libraries installed, it will be built with support for Lua plan files, which allow for mighty PDF transformations.
+- **podofomerge** - Merges two PDF files into onw.
+- **podofopdfinfo** - Provides some basic info about a PDF - metadata, page details, etc.
+- **podofotxt2pdf** - Converts a text file to a PDF
+- **podofotxtextract** - A tool that extracts all text from a PDF file. Works only for simple PDFs at the moment.
+- **podofouncompress** - Removes all compression filters from a PDF file. This is useful for debugging existing PDF files. You might also find PoDoFoBrowser, distributed separately, to be useful if you're using this tool a lot.
+
+Additionally there is the external tool PoDoFoBrowser which is not included in this package, but can be downloaded from the PoDoFo webpage. PoDoFoBrowser is a Qt application for browsing and modifying the objects in a PDF file, examining the structure of PDF files, and manipulating PDF streams. It is very useful if you want to look on the internal structure of PDF files.
+
+As of version 0.7 PoDoFo is available for Unix, Mac OS X and Windows platforms.
+
+## Where can I can get it?
+
+PoDoFo is available on the internet: [podofo.sf.net](http://podofo.sf.net)
```

Comment:
I appreciate not mixing the format transition with content changes, but we probably want to point at https://github.com/podofo/podofo/ now.